.gitignore

```diff
@@ -1,3 +1,29 @@
+# Ignore directories generated by Composer
+/drush/contrib/
+/vendor/
+/docroot/core/
+/docroot/modules/contrib/
+/docroot/themes/contrib/
+/docroot/profiles/contrib/
```

Comment:
Varbase is located in `/docroot/profiles` directly 
We should move that too
Better in `/docroot/profiles/contrib/`